stanford/playbooks/users.yml

```diff
@@ -1,4 +1,4 @@
-- hosts: "tag_Deployment_{{ COMMON_DEPLOYMENT }}"
+- hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_Deployment_{{ COMMON_DEPLOYMENT }}"
```

Comment:
Does this change mean the other user plays need to define `CLUSTER_NAME` as well?

Reply:
More like, if you want to target another cluster, you use:

```sh
./users.yml -e CLUSTER_NAME=edxapp
```